addons/l10n_sa/views/report_invoice.xml

```diff
@@ -102,11 +107,18 @@
                 إجمالي قيمة الفاتورة شامل ضريبة القيمة المضافة
             </strong>
         </xpath>
+
         <xpath expr="//div[@name='invoice_date']//span" position="before">
             <span t-if="o.l10n_sa_confirmation_datetime" t-field="o.l10n_sa_confirmation_datetime"/>
         </xpath>
         <xpath expr="//div[@name='invoice_date']//span[@t-field='o.invoice_date']" position="attributes">
             <attribute name="t-if">not o.l10n_sa_confirmation_datetime</attribute>
         </xpath>
+        <xpath expr="//p[@name='payment_term']//div" position="attributes">
```

Comment:
Why do you add this ? From what I see, this is always only one line on the invoice, did you find some exceptions ?